collector/app/views.py

```diff
@@ -1,7 +1,57 @@
 """This module provides views for collector app."""
 
+from aiohttp import web
+from core.jwt import decode_jwt
 
-from aiohttp.web import RouteTableDef
+from app.utils.monobank import parse_transaction_response
 
 
-routes = RouteTableDef()
+routes = web.RouteTableDef()
+
+
+@routes.view("/monobank/{token}")
+class MonobankWebhook(web.View):
+    """Class that represent functionality to work with monobank webhook."""
+
+    def _check_permission(self):
+        """Return user_id if provided token is correct."""
+        token = self.request.match_info["token"]
+        secret = self.request.app["constants"]["MONOBANK_WEBHOOK_SECRET"]
+        decoded_token = decode_jwt(token, secret)
+        if decoded_token:
```

Comment:
It's probably should be "if not decoded_token:".

Reply:
Also, I think better approach will be to raise a JWT exception in the "decode_jwt" function instead of returning none. If I'm not mistaken, jwt library raises these exceptions if it's something wrong with the token. But it's up to you.

Reply:
Yeap, but it's lives in core, which was realized and I don't want to play with it for now.